client/db_test.go

```diff
@@ -288,3 +290,46 @@ func TestDebugName(t *testing.T) {
 		return nil
 	})
 }
+
+func TestCommonMethods(t *testing.T) {
+	types := []reflect.Type{
+		reflect.TypeOf(&client.DB{}),
+		reflect.TypeOf(&client.Tx{}),
+		reflect.TypeOf(&client.Batch{}),
+		reflect.TypeOf(client.DB{}.B),
+	}
+	blacklist := map[string]bool{
```

Comment:
ah, right you are. one more nit: can you add something that makes sure things on the blacklist exist on at least one of `types`?